pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "apiron"
-version = "9.0.0"
+version = "9.1.0"
```

Comment:
@daneah Was re-familiarizing myself on the release process for this project and I see that we cut releases separate from the underlying code changes. Shall I revert this version bump (and adjust the changelog), and then open a separate PR to do the release?

Reply:
I think this is worth getting a release out right away for, so I have no qualms leaving this as-is.